rpcs3/Emu/Cell/Modules/cellGame.cpp

```diff
@@ -786,14 +786,78 @@ error_code cellGameCreateGameData(vm::ptr<CellGameSetInitParams> init, vm::ptr<c
 
 error_code cellGameDeleteGameData(vm::cptr<char> dirName)
 {
-	cellGame.todo("cellGameDeleteGameData(dirName=%s)", dirName);
+	cellGame.warning("cellGameDeleteGameData(dirName=%s)", dirName);
 
 	if (!dirName)
 	{
 		return CELL_GAME_ERROR_PARAM;
 	}
 
-	return CELL_OK;
+	const std::string name = dirName.get_ptr();
+	const std::string dir = vfs::get("/dev_hdd0/game/"s + name);
+
+	const auto prm = g_fxo->get<content_permission>();
+
+	auto remove_gd = [&]() -> error_code
+	{
+		if (Emu.GetCat() == "GD" && Emu.GetTitleID() == name)
```

Comment:
I don't think this is correct.
Just because game data shares the same ID, doesn't mean it's the boot for.
I think Emu.GetDir() would make more sense here.
Also, the comment below should state that a boot patch was meant.

Reply:
fixed.